pandas-stubs/core/strings.pyi

```diff
@@ -160,7 +167,7 @@ class StringMethods(NoNewAttributesMixin, Generic[T, _TS, _TM, _TS2]):
     def count(self, pat: str, flags: int = ...) -> Series[int]: ...
     def startswith(self, pat: str | tuple[str, ...], na: Any = ...) -> Series[bool]: ...
     def endswith(self, pat: str | tuple[str, ...], na: Any = ...) -> Series[bool]: ...
-    def findall(self, pat: str, flags: int = ...) -> Series: ...
+    def findall(self, pat: str, flags: int = ...) -> UnknownSeries: ...
```

Comment:
this can be narrowed down actually, it should be `Series[list[str]]`, looks like that pattern is used elsewhere (e.g. `rsplit`)

Reply:
OK - go for it